agent/lib/ipaddr_helpers.rb

```diff
@@ -0,0 +1,48 @@
+class IPAddr
```

Comment:
Yes, the stdlib `IPAddr` is really inadequate. I figure these should be replaced with the [`ipaddress`](https://github.com/ipaddress-gem/ipaddress) gem.

The worst part is the crazy-looking `IPAddr` operations on the server, like

```ruby
(IPAddr.new(self.grid.subnet) | self.node_number).to_s
```

This would be much nicer with the agent IPAddr patching (`IPAddr.new(node_info['grid']['subnet'])[node_number]`), but the way the kontena repo is structured, there's no easy way to share code between across all three sub-components...